tools/deploy.py

```diff
@@ -148,26 +155,26 @@ def main():
 
             ir_files.append(save_path)
 
-    # calib data
-    calib_filename = get_calib_filename(deploy_cfg)
-    if calib_filename is not None:
-        calib_path = osp.join(args.work_dir, calib_filename)
-
-        create_process(
-            'calibration',
-            create_calib_table,
-            args=(calib_path, deploy_cfg_path, model_cfg_path,
-                  checkpoint_path),
-            kwargs=dict(
-                dataset_cfg=args.calib_dataset_cfg,
-                dataset_type='val',
-                device=args.device),
-            ret_value=ret_value)
-
     backend_files = ir_files
     # convert backend
     backend = get_backend(deploy_cfg)
     if backend == Backend.TENSORRT:
+        # calib data
+        calib_filename = get_calib_filename(deploy_cfg)
+        if calib_filename is not None:
+            calib_path = osp.join(args.work_dir, calib_filename)
+
+            create_process(
+                'trt_calibration',
```

Comment:
Errr, strictly, it is designed to support all backend native PTQ. It will create a h5py file without PyTorch or any other dependency(It works like the QuantizationImageDataset you added, except it will serialize the data into a file.). Can we add a flag in config to switch native PTQ and our PTQ?

Reply:
get calibration data has no conflict with PTQ, so revert it.